devenv/sync.py

```diff
@@ -72,13 +72,18 @@ def run_procs(
 
 
 # Temporary, see https://github.com/getsentry/sentry/pull/78881
-def check_minimum_version(minimum_version: str):
+def check_minimum_version(minimum_version: str) -> bool:
     version = importlib.metadata.version("sentry-devenv")
 
     parsed_version = tuple(map(int, version.split(".")))
     parsed_minimum_version = tuple(map(int, minimum_version.split(".")))
 
-    if parsed_version < parsed_minimum_version:
+    return parsed_version >= parsed_minimum_version
+
+
+def main(context: dict[str, str]) -> int:
+    minimum_version = "1.13.0"
+    if not check_minimum_version(minimum_version):
```

Comment:
aren't we able to remove this now that `devenv` itself does this check too?

Reply:
there's no way of knowing if everyone is synced to at least 1.13.0 yet - don't want to risk more breakage

Reply:
makes sense -- I guess we can wait a few weeks and then remove it 👍

Reply:
yup thats the plan!